setup.cfg.rej

```diff
@@ -0,0 +1,23 @@
+diff a/setup.cfg b/setup.cfg	(rejected hunks)
```

Comment:
@daavoo looks like there is an issue with the cruft PR action? It's committing rejected/conflicted change files

Reply:
Not sure how to handle it without manual intervention. Ideally would like to just commit the changes and resolve conflicts on github UI but `cruft` doesn't seem to have an option to force

Reply:
I think this will be a common issue for us with the way the template updates are automated right now. Since we use dependabot in both the template and the dvc-module repos, we are going to hit these conflicts regularly any time that dependabot has already updated something in the dvc-module repo.

It might be better for us to just not use the automation, and manually run cruft in the relevant modules whenever there is a feature change in the template repo that actually needs to be applied (vs dependency version updates that can also just be done via dependabot)

Reply:
I had enabled dependabot on the template itself but haven't removed it from the generated repos.

I think makes more sense to only have dependabot in the original template and merge the updates via cruft. Removing dependabot from the generated repos should prevent this conflicts in the future. Other conflicts will have same issue, though

Reply:
The problem there is that the generated repos will presumably have their own specific dependencies that do not belong in the template repo (dvc-task depending on celery + other celery related packages for example)

Reply:
Maybe we could add all dependencies that are set in the template to the ignore list in the generated repo's templated dependabot config. So shared dependencies would only be updated through the `iterative/py-template`'s dependabot config + cruft.

so `{{cookiecutter.project_name}}/.github/dependabot.yml` would need to contain `ignore` entries for everything listed in the template's `setup.cfg` (but this would not affect the py-template dependabot setup)

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore

Reply:
That makes sense! will do